src/documentCreator/contexts/FormDataContext/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement, ReactNode } from "react";
+import { issueDocument } from "@govtechsg/tradetrust-schema";
+import { WrappedDocument } from "@govtechsg/open-attestation";
+import { Document } from "@govtechsg/decentralized-renderer-react-components";
+
+interface FormDataState {
+  documentsList: Array<Document>;
+  setDocumentsList: (documentsList: Document[]) => void;
+  wrappedDocument: WrappedDocument<any> | null;
```

Comment:
```suggestion
  wrappedDocument: WrappedDocument<any> | undefined;
```

Will personally prefer `undefined` over `null` if you mean for it to be an intermediate state from unassigned to assigned. Null is an actual value. The topper to that is that using undefined, later you can use WrappedDocument?